observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemSchema.java

```diff
@@ -5,37 +5,61 @@
 package org.wildfly.extension.micrometer;
 
 import static org.jboss.as.controller.PersistentResourceXMLDescription.factory;
+import static org.wildfly.extension.micrometer.MicrometerConfigurationConstants.OTLP_REGISTRY;
+import static org.wildfly.extension.micrometer.MicrometerConfigurationConstants.PROMETHEUS_REGISTRY;
+import static org.wildfly.extension.micrometer.prometheus.PrometheusRegistryDefinitionRegistrar.RESOURCE_REGISTRATION;
+
+import java.util.EnumSet;
+import java.util.Set;
 
 import org.jboss.as.controller.PersistentResourceXMLDescription;
 import org.jboss.as.controller.PersistentSubsystemSchema;
 import org.jboss.as.controller.SubsystemSchema;
 import org.jboss.as.controller.xml.VersionedNamespace;
+import org.jboss.as.version.Stability;
 import org.jboss.staxmapper.IntVersion;
+import org.wildfly.extension.micrometer.otlp.OtlpRegistryDefinitionRegistrar;
+import org.wildfly.extension.micrometer.prometheus.PrometheusRegistryDefinitionRegistrar;
 
 public enum MicrometerSubsystemSchema implements PersistentSubsystemSchema<MicrometerSubsystemSchema> {
-    VERSION_1_0(1, 0), // WildFly 28
-    VERSION_1_1(1, 1), // WildFly 29.0.0.Alpha1
+    VERSION_1_0(1, 0, Stability.DEFAULT), // WildFly 28
+    VERSION_1_1(1, 1, Stability.DEFAULT), // WildFly 29.0.0.Alpha1
+    VERSION_1_1_COMMUNITY(1, 1, Stability.COMMUNITY) // WildFly 35
```

Comment:
@bstansberry @pferraro I'm not sure why we would want to bump the version to this instead of 2.0, but if you're ok with it, I'll trust you. :)

Reply:
I'm not sure I follow this. Is there a reason we didn't upgrade to 2.0?

Reply:
I *think* what we settled on was adding new lower stability stuff to an existing model results in keeping the same version, different stability.

We have a different problem here though. This PR does 2 things:

1) Rearranges the model for OTLP. That has nothing to do with community stability and requires a new 2.0, Stability.DEFAULT.
2) Adds a new 'community' child resource, and thus a 2.0 Stability.COMMUNITY.

Perhaps we could skip the 2.0 DEFAULT bit if we could absolutely prove (via regression tests) that starting a 'default' stability server with a 1.1 config and modifying an existing 1.1 model attribute results in a persisted config that can be successfully rebooted. IOW, any translation that would introduce the new model structure with the registry=otlp resource would not result in that structure being persisted into xml.

Reply:
FTR: I have adjusted the version numbers here. The first commit in this PR bumped things to 2.0, then, after some discussion, the second commit bumped it back down to 1.1. I reverted that change and added the 2.0 Community as described here.